### PR TITLE
Created Buttons and Badges

### PR DESCRIPTION
**Pull request by sanu29**
image
image
image
image

### CODE REVIEW

File: components/badges.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
+.badge{
+    display         : inline ;
+    margin          : 4px;
+    line-height     : 48px;
```
Comment (zoef-yt) [inline components/badges.css]: use rems

File: components/badges.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    background-color: var(--dark);
+}
+.badge-success{
+    background-color: var(--success);
+}
+
+/* badge on avtar */
+.badge-online{
+    background-color:var(--success);
+    border          : solid 2px #fff;
```
Comment (zoef-yt) [inline components/badges.css]: use var for colour

File: components/badges.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    height          : 1rem;
+    width           : 1rem;
+    bottom          : -5px;
+    right           : 5px;
+}
+.badge-buzy{
+    background-color:var(--danger);
+    border          : solid 2px #fff;
```
Comment (zoef-yt) [inline components/badges.css]: use var for colour

File: components/badges.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    height          : 1.3rem;
+    width           : 1.3rem;
+    bottom          : 2px;
+    right           : 7px;
+    text-align      : center;
+    color           : #fff;
```
Comment (zoef-yt) [inline components/badges.css]: use var for colour

**Review by developer-anand — APPROVED** [review]
Approved

File: components/badges.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+}
+.badge-offline{
+    background-color:var(--gray);
+    border          : solid 2px #fff;
```
Comment (zoef-yt) [inline components/badges.css]: use var for colour

File: components/button.css [hunk 6]
```diff
@@ -0,0 +1,192 @@
+.btn{
+    margin          : 0.5rem;
+    padding         : 0.8rem;
```
Comment (zoef-yt) [inline components/button.css]: use px maybe?

File: components/button.css [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+}
+.btn-icon:hover{
+    box-shadow      : 1px 1px 10px 1px var(--gray);
+}
+.btn-margin-blur{
+    box-shadow      :0px 0px 13px 1px var(--success);
+}
+.btn-margin{
+    border          :solid 2px white;
+    box-shadow      :0px 0px 1px 3px #9f091d;
```
Comment (zoef-yt) [inline components/button.css:166]: use var for colour

File: components/button.css [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+}
+.btn-margin-shadow{
+    box-shadow      :1px 1px 5px 0px rgba(0,0,0,0.6), -5px 2px 50px 0px #037cd5;
```
Comment (zoef-yt) [inline components/button.css]: use var for colour

File: components/button.css [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+}
+.badge-action-button{
+    background-color:var(--info);
+    color           : var(--light);
+    border          : solid 2px #fff;
```
Comment (zoef-yt) [inline components/button.css]: use var for colour

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by zoef-yt — APPROVED** [review]
Clean code. Good to merge.

**Pull request merged**